homeassistant/components/airzone/binary_sensor.py

```diff
@@ -15,6 +16,7 @@
 )
 
 from homeassistant.components.binary_sensor import (
+    DEVICE_CLASS_BATTERY,
```

Comment:
DEVICE_CLASS* are deprecated as of 2021.12, use the BinarySensorDeviceClass enum instead. I suggest that at least for newly used values you use the new enum, existing entries could be changed in a separate PR.

Reply:
Should be fixed in 33a2023cc4e41b575a32b7a2ad9893b93d8b518d